README.md

```diff
@@ -122,6 +125,7 @@ Options:
 -p, --prepend-filename      Prepend filename to the package name in the report.  Helps distinguish between multiple runs/diff browser/same test [default: false]
 -r, --relative-screenshots  Convert screenshot paths from absolute to relative to output file. [default: false]
 -b, --browser               If true generated report will be opened in the browser. [default: true]
+-f, --browser-fail   				If true, generated report will only open in browser on test failure. [default: false]
```

Comment:
This PR is probably not going to be accepted, BUT you've got tabs instead of spaces here :)

Reply:
What makes you say that? I don’t see any tab characters.